src/index.js

```diff
@@ -0,0 +1,10 @@
+import WASI from "./wasi.js"
+import { File } from "./fs_core.js"
+import { PreopenDirectory } from "./fs_fd.js"
+
+module.exports = {
+    WASI,
+    File,
+    PreopenDirectory,
+}
```

Comment:
What else should we expose here? Also open to namespace any of these objects if that makes more sense.

Reply:
I think strace should be exported too for debugging.